src/screens/home/PortfolioComponent.tsx

```diff
@@ -24,11 +29,14 @@ export const PortfolioComponent = ({
   showTotalCard = true,
   style,
 }: Props) => {
+  const chainId = useAppSelector(selectChainId)
```

Comment:
it's better if we use `getCurrentChainId` since anytime we switch networks we modify this, it will make the `chainId` fetching more consistent and it can be used everywhere.

Reply:
done